src/App.tsx

```diff
@@ -1,14 +1,46 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
```

Comment:
Consider removing the eslint-disable comment and fixing the accessibility issue by adding a label to the control. ♿